openapi-diff/src/core/OpenApiDiff.Core/Logging/ObjectPath.cs

```diff
@@ -66,10 +85,13 @@ public static string FileNameNorm(string fileName)
             => fileName.Replace("\\", "/");
 
         // https://tools.ietf.org/html/draft-ietf-appsawg-json-pointer-04
-        public string JsonPointer(IJsonDocument t) => CompletePath(t.Token)
-            .Select(v => v.name?.Replace("~", "~0")?.Replace("/", "~1"))
-            .Aggregate(FileNameNorm(t.FileName) + "#", (a, b) => a == null || b == null ? null : a + "/" + b);
-            
+        public string JsonPointer(IJsonDocument t)
+        {
+            var result = CompletePath(t.Token)
```

Comment:
what format is completePath that we need to make these replacements

Reply:
Returns a sequence of property names, including the "#" string.